apps/devhub/tests/test_views.py

```diff
@@ -1207,6 +1212,23 @@ def test_delete_and_recreate_credentials(self):
         assert new_key.key != old_key.key
         assert new_key.secret != old_key.secret
 
+    def test_retreive_credentials(self):
+        client = Client(enforce_csrf_checks=True)
+        assert client.login(username='regular@mozilla.com', password='password')
+
+        response = client.get(reverse('devhub.api_creds'),
+                                   {'key': 'abc123'})
+        assert response.status_code == 200
+        response_json = json.loads(response.content)
+        assert response_json['secret'] == 'supersekret'
+
+    def test_retreive_credentials_404(self):
```

Comment:
this test name should indicate what it is you're testing or else you need to add a docstring. For example, a better name might be `test_cannot_get_other_users_credentials()`